Orange/statistics/util.py

```diff
@@ -562,7 +562,7 @@ def nanmode(x, axis=0):
 def unique(x, return_counts=False):
     """ Equivalent of np.unique that supports sparse or dense matrices. """
     if not sp.issparse(x):
-        return np.unique(x, return_counts=return_counts)
+        return np.unique(np.asarray(x), return_counts=return_counts)
```

Comment:
Do you remember who called unique?

Reply:
in owlearnerwidget there's a `check_data` method, that makes sure that (among other things) Y isn't constant. I can move the computation up there instead if that makes more sense.